packages/headless/src/controllers/core/generated-answer/headless-core-generated-answer.ts

```diff
@@ -84,6 +87,11 @@ export interface GeneratedAnswer extends Controller {
    * @param details - Details on why the generated answer was not relevant.
    */
   sendDetailedFeedback(details: string): void;
+  /**
```

Comment:
Should the old function be annotated as `@deprecated` in JS doc ?

Another way to go about this would be to change `sendFeedback(feedback: GeneratedAnswerFeedback | GeneratedAnswerFeedbackV2)`, and at runtime, detect the shape of the feedback object (is it v1 or v2) and dispatch the correct action from there.

Reply:
I guess, just in general, the `v2` name on a public facing function is problematic, and will need to coordinate to deprecate in the next major version.

epic for V3: https://coveord.atlassian.net/browse/KIT-2157

Reply:
@olamothe Yes i love the first solution, it's way better, I will update according to thaat!